Makefile

```
@@ -388,6 +388,10 @@ help:
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 .PHONY: help
 
+clean: ## restore repository state as it was freshly cloned
+	git clean -idx
```

Some extremists would propose the following:

```suggestion
	git clean -fdx
```

But your moderate suggestion is perfectly fine :sweat_smile:

---

I buy it !

---

Will merge this PR as is. We'll see if we must be more extremist or not! :smiley_cat: